backend/api/v1/endpoints/research.py

```
@@ -144,12 +163,12 @@
         if not paper:
             raise HTTPException(status_code=404, detail="Paper not found")
 
-        file_path = paper.file_path  # Ensure this stores the absolute or relative file path
+        file_path = f"uploads/research_papers/{paper.file_path}"  # Ensure this stores the absolute or relative file path
         
         if not os.path.exists(file_path):
             raise HTTPException(status_code=404, detail="File not found on the server")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)